DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: easyr
 Type: Package
 Title: Helpful Functions from Oliver Wyman Actuarial Consulting
-Version: 0.5-11
+Version: 0.6-01
```

Comment:
Please use 0.5-11. We'll only move to 0.6 if there is a major change. And given the specific nature of easyr, that might never happen.